homeassistant/components/binary_sensor/strings.json

```diff
@@ -156,8 +160,8 @@
       "on": "Plugged in"
     },
     "presence": {
-      "off": "[%key:component::device_tracker::state::_::not_home%]",
-      "on": "[%key:component::device_tracker::state::_::home%]"
+      "off": "Away",
+      "on": "Home"
```

Comment:
here as well :)

Reply:
my bad, this was accidental changed during solving https://github.com/home-assistant/core/pull/53945#discussion_r689853224